test/middleware/service.availability.middleware.spec.unit.ts

```diff
@@ -1,9 +1,36 @@
 jest.mock("ioredis");
+jest.mock("../../src/middleware/authentication.middleware");
```

Comment:
now the landing page uses the middleware, the test needs the middleware session stuff mocked to pass